MeetU/MeetU/Views/Account/Login.cshtml

```diff
@@ -48,11 +48,18 @@
 @section Scripts {
     @Scripts.Render("~/bundles/jqueryval")
     <script>
-        $(".js-login-jump-icon").click(function () {
+        if ($(".validation-summary-errors").length > 0) {
+            scrollLogin(0)
+        } else {
+            setTimeout(scrollLogin, 1500, 900);
```

Comment:
The extra parameter is an additional parameter passed to the function.
_setTimeout(scrollLogin, 1500, 900)_  means executing _scrollLogin(900)_ in 1500 milliseconds.
![qq 20160418014547](https://cloud.githubusercontent.com/assets/12708531/14587441/4e953246-0507-11e6-8d88-09755043ac33.jpg)

Just noticed this is not compatible in IE, will improve it